src/MSBuildTaskHost/MSBuildTaskHost.csproj

```diff
@@ -62,6 +62,7 @@
       <Link>ITaskItem2.cs</Link>
     </Compile>
     <Compile Include="..\Framework\AssemblyUtilities.cs" />
+    <Compile Include="..\Framework\ResponseFileUsedEventArgs.cs" />
```

Comment:
This seems a bit surprising. Under what circumstances is the event used in MSBuildTaskHost.exe? In general, new stuff doesn't go there because it exists only for compat with ancient tasks.

Reply:
Oh, is it because it's touched in LogMessagePacketBase? That would make sense and be fine.

Reply:
Yup. Otherwise it won't build